docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,43 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 14 October 2022
+
+Frontend version: 1.0.0-20221013.0247 | Backend version: 1.32.4-221013.0846
+
+**New features** - **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Tenant admins can now create and update applications using APIs.</summary>
+
+Tenant Admins can now create and update applications using APIs. For more information, refer to [TechPass Automation API](https://docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/Tenant).
+
+This new feature complements the existing functionality to [create and update applications through the TechPass portal](https://docs.developer.tech.gov.sg/docs/techpass-tenant-guide/applications?id=registering-an-app).
+
+</details>
+
+**Fixes** - **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Terminated admins will not be notified of expiring or expired secrets and certificates.</summary>
+
+Terminated admins will no longer be notified when an application's secret and certificate are nearing expiry or expired.
+
+</details>
+
+**Changes** - **Frontend** and **Backend**
+
+<details>
+<summary style="font-size:20px;font-weight:bold">Security enhancements.</summary>
+
+We have made some changes to improve the security of your applications. When you create and update applications, ensure the **Homepage URL** and **Redirect URL** are in the following formats: ```http://localhost``` or ```https://```.
```

Comment:
Siva can you please reflect this format in the tenant guide as well

Reply:
Sure will do it.